cclib/parser/data.py

```diff
@@ -141,31 +143,48 @@ class ccData:
        "moenergies":       Attribute(list,             'energies',                    'properties:orbitals'),
+       "molecular_mass":   Attribute(float,            'molecular mass',              'properties'),
        "moments":          Attribute(list,             'total dipole moment',         'properties'),
        "mosyms":           Attribute(list,             'molecular orbital symmetry',  'properties:orbitals'),
        "mpenergies":       Attribute(numpy.ndarray,    'moller plesset',              'properties:energy'),
        "mult":             Attribute(int,              'multiplicity',                'properties'),
        "natom":            Attribute(int,              'number of atoms',             'properties'),
        "nbasis":           Attribute(int,              'basis number',                'properties:orbitals'),
        "nmo":              Attribute(int,              'MO number',                   'properties:orbitals'),
-       "nmrtensors":       Attribute(dict,             'NMR chemical shielding tensors', 'properties:nmr'),
+       "nmr_anis":         Attribute(numpy.ndarray,    'NMR anisotopic tensors',      'properties:NMR'),
+       "nmr_eigen":        Attribute(numpy.ndarray,    'NMR eigenvalues',             'properties:NMR'),
+       "nmr_iso":          Attribute(numpy.ndarray,    'NMR isotopic tensors',        'properties:NMR'),
+       "nmrtensors":       Attribute(dict,             'NMR chemical shielding tensors', 'properties:NMR'),
        "nocoeffs":         Attribute(numpy.ndarray,    'TBD',                         'N/A'),
        "nooccnos":         Attribute(numpy.ndarray,    'TBD',                         'N/A'),
-       "nsocoeffs":         Attribute(list,    'TBD',                         'N/A'),
-       "nsooccnos":         Attribute(list,    'TBD',                         'N/A'),
+       "nsocoeffs":        Attribute(list,             'TBD',                         'N/A'),
+       "nsooccnos":        Attribute(list,             'TBD',                         'N/A'),
+       "oniom_energy":     Attribute(list,             'ONIOM energy',                'optimization:ONIOM'),
+       "oniom_energy_fin": Attribute(float,            'ONIOM energy',                'properties:energy'),
        "optdone":          Attribute(list,             'done',                        'optimization'),
        "optstatus":        Attribute(numpy.ndarray,    'status',                      'optimization'),
+       "point_group":      Attribute(str,              'symmetry point group',        'properties:rotational'),
        "polarizabilities": Attribute(list,             'polarizabilities',            'N/A'),
        "pressure":         Attribute(float,            'pressure',                    'properties'),
+       "roconst":          Attribute(numpy.ndarray,    'rotational constants',        'properties:rotational'),

```

Comment:
This is currently in the master branch with a different organization, so im looking into how to resolve this now.

Reply:
https://github.com/jvalegre/cclib/pull/3 this pull request will merge with main branch and make the changes necessary for the rotational constant parsing that was introduced during the course of this branches development.